client/gutenberg/extensions/slideshow/settings.js

```diff
@@ -0,0 +1,71 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
+
+export const settings = {
+	name: 'slideshow',
+	prefix: 'jetpack',
```

Comment:
Where is `prefix` used?

Reply:
This part is being used in when we add the div class. I think

Reply:
`prefix` is used here https://github.com/Automattic/wp-calypso/blob/master/client/gutenberg/extensions/shared/frontend-management.js#L18. When we switched from `registerBlockType()` to `registerJetpackBlock()`, we dropped the `jetpack/` prefix in block names since `registerJetpackBlock()` automatically applies the prefix. In order to accurately produce the block className (`wp-block-jetpack-slideshow`) the prefix needs to be provided.